### PR TITLE
[12.0][IMP] Variables for ICMSSN

### PR DESCRIPTION
**Pull request by ygcarvalh**
Good afternoon people.

This PR adds some variables that were missing for the calculation of the taxation of Simples Nacional and was affecting the generation of the fields of the NF-e when the CSOSN was worth 500 or 900.

I also put these fields in the view and the conditions for them to appear only when they are needed.

### CODE REVIEW

**Comment by ygcarvalh** [comment]
@mileo @luismalta

**Review by gabrielcardoso21 — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by marcelsavegnago — CHANGES_REQUESTED** [review]
@ygcarvalh  Pode verificar os erros do travis por favor ?

**Comment by marcelsavegnago** [comment]
image

cc @mileo @ygcarvalh @luismalta @DiegoParadeda @gabrielcardoso21

**Comment by ygcarvalh** [comment]
@marcelsavegnago @mileo se puderem revisar novamente.

Outro ponto é que essa ainda semana vou fazer um novo PR para calcular essas novas variáveis que foram criadas.

**Review by marcelsavegnago — APPROVED** [review]
lgtm

**Review by mileo — APPROVED** [review]

File: l10n_br_fiscal/models/document_fiscal_line_mixin.py [hunk 1]
```diff
@@ -437,10 +449,34 @@ def _operation_domain(self):
...
+    icmsfcp_percent_wh = fields.Float(string="FCP WH %")
+
+    # vFCPSTRet - Valor do FCP retido anteriormente por ST
+    icmsfcp_value_wh = fields.Monetary(string="FCP WH Value")
+
+    # pRedBCEfet - Percentual de redução da base de cálculo efetiva
+    effective_base_percent = fields.Float(string="Effective Base %")
+
+    # vBCEfet - Valor da base de cálculo efetiva
+    effective_base_value = fields.Monetary(string="Effective Base Value")
```
Comment (mileo) [inline l10n_br_fiscal/models/document_fiscal_line_mixin.py]: Esses campos não deveriam iniciar com icms_? @ygcarvalh

File: l10n_br_fiscal/models/document_fiscal_line_mixin.py [hunk 2]
```diff
@@ -437,10 +449,34 @@ def _operation_domain(self):
...
+
+    # vBCEfet - Valor da base de cálculo efetiva
+    icms_effective_base_value = fields.Monetary(string="Effective Base Value")
+
+    # pICMSEfet - Alíquota do ICMS Efetiva
+    icms_effective_percent = fields.Float(string="ICMS Effective %")
+
+    # vICMSEfet - Valor do ICMS Efetivo
+    icms_effective_value = fields.Monetary(string="ICMS Effective Value")
+
```
Comment (mileo) [inline l10n_br_fiscal/models/document_fiscal_line_mixin.py]: Ainda não esta legal, tenta seguir a nomemclatura atual, por exemplo:

https://github.com/OCA/l10n-brazil/blob/4112414667b72b023a6b92ed91aa929899d70353/l10n_br_fiscal/models/document_fiscal_line_mixin.py#L340-L350

Comment (mileo) [reply l10n_br_fiscal/models/document_fiscal_line_mixin.py]: icms_effective_base_value -> icms_effective_base
icms_effective_percent ok
icms_effective_base_percent -> icms_effective_reducion
icms_effective_value ok

**Review by mileo — APPROVED** [review]

**Review by renatonlima — APPROVED** [review]

**Comment by mileo** [comment]
/ocabot merge patch

Tentando novamente!

**Comment by mileo** [comment]
O erro foi no webservice do correios que parece estar fora do ar: 
https://travis-ci.com/github/OCA/l10n-brazil/jobs/499482447#L3969

**Pull request merged**